scripts/garage

```diff
@@ -8,23 +9,27 @@ from garage.tf.experiment import LocalTFRunner
 
 @click.group()
 def cli():  # noqa: D103
+    # pylint: disable=missing-function-docstring
```

Comment:
why not just add a one-line docstring? It's as long as this pragma to disable it lol

Reply:
And most likely shorter!